### ISSUE TITLE
How to configure admin for custom user?

### ISSUE THREAD

**Issue by martasd** [jambonsw/django-improved-user#115]
Hi,

there is a nice description on how to create a custom user with mixins here:

https://django-improved-user.readthedocs.io/en/latest/create_custom_user_with_mixins.html

Could you please add to the documentation what is the recommended way to configure the custom user admin? I've tried with improved user's `UserAdmin`, but it does not work since I do not want to use `short_name` and `full_name`. Could you please explain the best way to do so?

**Comment by jambonrose** [comment]
Hi @martasd :

I wanted to find a way to provide an admin base for developers using the extension or replacement method, but I've reached a dead end. Unfortunately, inheriting admin configurations is messy, and even composing different admin configurations proved to be tricky and error prone. Due to these issues, my formal recommendation is that developers supply their own `UserAdmin` classes.

I've added a [section in the documentation detailing expected usage.](https://django-improved-user.readthedocs.io/en/latest/admin_usage.html)

I'm taking the liberty of closing this issue, but I'm happy to discuss expanding the documentation or trying to add a feature on this front. Feel free to discuss below or to open a new issue.
